yarn/tests/test_unit.py

```diff
@@ -0,0 +1,31 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.yarn import YarnCheck
+
+from .common import YARN_CONFIG
+
+
+@pytest.mark.parametrize(
+    'tags, expected_tags',
+    [
+        pytest.param("test:example", ["app_test:example"], id='tag_key_value'),
+        pytest.param("test", ["app_test"], id='tag_value_only'),
+        pytest.param(
+            "script_name:test_script,value_only_tag,user_email:test_email",
+            ["app_script_name:test_script", "app_value_only_tag", "app_user_email:test_email"],
+            id='both_tag_key_and_value_only',
```

Comment:
Can we add tests for multiple `key:value` tags and multiple `value` tags only as well?